prow/plugins/lgtm/lgtm.go

```diff
@@ -330,12 +331,25 @@ func handle(wantLGTM bool, config *plugins.Configuration, ownersClient repoowner
 }
 
 func stickyLgtm(log *logrus.Entry, gc githubClient, config *plugins.Configuration, lgtm *plugins.Lgtm, author, org, repo string) bool {
-	if lgtm.StickyForTrustedAuthors {
-		triggerOpt := config.TriggerFor(org, repo)
-		if ok, merr := trigger.TrustedUser(gc, triggerOpt, author, org, repo); merr == nil && ok {
-			return true
-		} else if merr != nil {
-			log.WithError(merr).Errorf("Failed to check if author is a collaborator.")
+	if lgtm.StickyForTrustedAuthors && len(lgtm.TrustedAuthorTeam) > 0 {
+		if teams, err := gc.ListTeams(org); err == nil {
```

Comment:
@cjwagner you really think it is cheaper to a) list all teams in the org and then b) list all members in the team vs a single call to https://developer.github.com/v3/repos/collaborators/#review-a-users-permission-level?

Reply:
Aside from cpu/network cost and efficiency, these check different things. Both functionally and conceptually, it is easier to organize the trusted users in a team, instead of checking admin or write permissions.

Eventually we may restrict write permission to the merge bot only, but we still want to trust some contributors. Using a github team to control this is more flexible.